utilities_common/multi_asic.py

```diff
@@ -148,3 +148,13 @@ def multi_asic_args(parser=None):
     parser.add_argument('-n', '--namespace', default=None,
                         help='Display interfaces for specific namespace')
     return parser
+
+class MultiAsicDb(object):
```

Comment:
@abdosi do we really need this new class, I see that the Db object itself is enhanced to supports multi-asic ? https://github.com/Azure/sonic-utilities/blob/master/utilities_common/db.py#L7

Reply:
@judyjoseph as part of next PR i will be removing that file and will be using MultiAsicDb so that code remains consistent in multiasic.py

Reply:
@judyjoseph On second thought I will close my PR#https://github.com/Azure/sonic-utilities/pull/1312 as I feel that will create conflict for other outstanding PR's and use existing Db class and update this PR accordingly.

Thanks for pointing out.